packages/components/src/Autocomplete/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -80,14 +80,14 @@ exports[`renders correctly when invalid 1`] = `
   >
     <label
       className="label"
-      htmlFor="123e4567-e89b-12d3-a456-426655440043"
+      htmlFor="123e4567-e89b-12d3-a456-426655440044"
```

Comment:
@kingstonfung this is a flakey snapshot that _might_ be contributing to your issues. Try kicking these changes out of your PR and see if that helps?